documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,232 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+
+When you get to the step to recreate your cluster, you have two options:
+
+. Use _Option 1_ when you can recover all `KafkaTopic` resources.
++
+In this situation, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must therefore be recovered before the cluster is started so that the corresponding topics are not deleted by the Topic Operator.
+
+. Use _Option 2_ when you are unable to recover all `KafkaTopic` resources.
++
+This time you deploy your cluster without the Topic Operator and then redeploy it so that the Topic Operator can recreate the `KafkaTopic` resources from the corresponding topics.
```

Comment:
```suggestion
This time you deploy your cluster without the Topic Operator, delete the Topic Operator data in Zookeeper and then redeploy it so that the Topic Operator can recreate the `KafkaTopic` resources from the corresponding topics.
```